application/src/main/java/net/corda/v5/application/interop/binding/InteropAction.java

```diff
@@ -41,6 +44,10 @@ public T getResult() {
             return this.result;
         }
 
+        /**
+         * The result of an [InteropAction] that has been performed by the server.
```

Comment:
Replace all `[<CLASS_NAME>]` with `{@link <CLASS_NAME>}`, the former work in Kotlin only the latter is for Java. So here `* The result of an {@link InteropAction} that has been performed by the server.
Make sure that the links are clickable (in rare cases a linked class is not used directly in the current source code, then you need to import that class in order to use in a link).